docs/managed-tests/validation.md

```diff
@@ -0,0 +1,34 @@
+# Validation Tests
```

Comment:
Again, missing links here. This rewrite is arguably ok, as the original is perhaps a bit too verbose, but I'm still iffy.

Reply:
Let me know what you think of this one:
![image](https://github.com/user-attachments/assets/af7b4102-4f0e-43d1-b75c-d860853a4cbf)